celements-scheduler/src/main/java/com/celements/scheduler/classdefs/SchedulerJobClass.java

```diff
@@ -0,0 +1,86 @@
+package com.celements.scheduler.classdefs;
+
+import org.springframework.stereotype.Component;
+import org.xwiki.model.reference.ClassReference;
+
+import com.celements.model.classes.AbstractClassDefinition;
+import com.celements.model.classes.fields.ClassField;
+import com.celements.model.classes.fields.LargeStringField;
+import com.celements.model.classes.fields.StringField;
+import com.xpn.xwiki.XWikiConstant;
+
+@Component(SchedulerJobClass.CLASS_DEF_HINT)
+public class SchedulerJobClass extends AbstractClassDefinition
+    implements SchedulerJobClassDefinition {
+
+  public static final String DOC_NAME = "SchedulerJobClass";
+  public static final String XWIKI_SPACE = XWikiConstant.XWIKI_SPACE;
+  public static final String CLASS_DEF_HINT = XWIKI_SPACE + "." + DOC_NAME;
+  public static final ClassReference CLASS_REF = new ClassReference(XWIKI_SPACE, DOC_NAME);
+
+  public static final ClassField<String> FIELD_JOB_NAME = new StringField.Builder(
+      CLASS_REF, "jobName")
+          .prettyName("Job Name")
+          .size(60)
+          .build();
+
+  public static final ClassField<String> FIELD_JOB_DESCRIPTION = new LargeStringField.Builder(
+      CLASS_REF, "jobDescription")
+          .rows(10)
+          .prettyName("Job Description")
+          .size(45)
+          .build();
+
+  public static final ClassField<String> FIELD_JOB_CLASS = new StringField.Builder(
+      CLASS_REF, "jobClass")
+          .prettyName("Job Class")
+          .size(60)
+          .build();
+
+  public static final ClassField<String> FIELD_STATUS = new StringField.Builder(
+      CLASS_REF, "status")
+          .prettyName("Status")
+          .size(30)
+          .build();
```

Comment:
Perhaps it would be useful to type the status field as an enum.

This is possible with `CustomStringField` and the `EnumMarshaller`.

Possible status are at least `Normal` and `None`.